client/a8c-for-agencies/components/sidebar/header/index.tsx

```diff
@@ -9,15 +9,21 @@ import { setLayoutFocus } from 'calypso/state/ui/layout-focus/actions';
 import { getCurrentLayoutFocus } from 'calypso/state/ui/layout-focus/selectors';
 import getSelectedSiteId from 'calypso/state/ui/selectors/get-selected-site-id';
 import { AppState } from 'calypso/types';
-import A4ALogo from '../../a4a-logo';
+import A4ALogo, { LOGO_COLOR_PRIMARY, LOGO_COLOR_SECONDARY_ALT } from '../../a4a-logo';
 import ProfileDropdown from './profile-dropdown';
 
 type Props = {
 	forceAllSitesView?: boolean;
 	simple?: boolean;
 };
 
-const AllSitesIcon = () => <A4ALogo className="a4a-sidebar__all-sites-icon" size={ 32 } />;
+const AllSitesIcon = () => (
+	<A4ALogo
+		className="a4a-sidebar__all-sites-icon"
+		colors={ { primary: LOGO_COLOR_PRIMARY, secondary: LOGO_COLOR_SECONDARY_ALT } }
```

Comment:
[Non-blocker] Sorry for not noticing it earlier. The default value for the `primaryColor` is `LOGO_COLOR_PRIMARY`; I think we can remove it.